lib/corm/model.rb

```diff
@@ -138,6 +140,43 @@ def self.count
       execute(statements['count']).first['count'].to_i
     end
 
+    ##
+    # Find by keys.
+    # This `find` methods wants to be as flexible as possible.
+    #
+    # Unless a block is given, it returns an `Enumerator`, otherwise it yields
+    # to the block an instance of the found Cassandra entries.
+    #
+    # If no keys as passed as parameter, the methods returns (an Enumerator for)
+    # all the results in the table.
+    #
+    # If one or more keys are given, the **order matters**; since this methods
+    # will craft a CSQL query, the first key **must** be the `partition key`,
+    # while the followings are intended as the `clustering keys`... that can be
+    # more than one.
+    #
+    # If the keys passed as parameter are more than the defined by the table the
+    # query is not valid, it cannot be executed and an error is raised.
+    def self.find(*key_values, &block)
+      return to_enum(:find, *key_values) unless block_given?
+
+      field_names = []
+      Array(key_values).each_with_index do |value, index|
+        field_names << "#{primary_key.flatten[index]} = ?"
+      end
+
+      statement_find_key = "find_#{Digest::MD5.hexdigest(field_names.inspect)}"
```

Comment:
@zeroed don't hash with md5: if we use something like `"find_#{values}"`, where `values` is the hash passed to the method, it will be easy for the user checking the content of `statements` var.
I think it's really hard to understand/debug what is "find_0c88028bf3aa6a6a143ed846f2be1ea4" and from where it's coming from.